docker/Dockerfile.rpm-yum

```diff
@@ -71,6 +71,7 @@ RUN make PREFIX=${DIST_DIR} cmds
 
 WORKDIR $DIST_DIR/..
 COPY packaging/rpm .
+COPY deployments/systemd/* ${DIST_DIR}/
```

Comment:
Is there a reason that we don't use: 
```suggestion
COPY deployments/systemd/ ${DIST_DIR}/
```
Like we do for the other Dockerfiles?

Reply:
done